### ISSUE TITLE
缺少SendMessageTimeoutW和SendMessageTimeoutA

### ISSUE THREAD

**Issue by xmsz-stu** [waitingsong/node-win32-api#80]
https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-sendmessagetimeoutw

**Comment by laztheripper** [comment]
Yeah I'm also needing this, using just SendMessageW eventually causes deadlocks when something goes wrong and the receiving app isn't able to read the message. This eventually causes my entire app to stop working because the wrapper for SendMessageW just waits infinitely for a return.

**Comment by waitingsong** [comment]
`SendMessageTimeoutW()` , `SendMessageTimeoutW()` added
